### ISSUE TITLE
v1.6.1: missing train/loss in progress bar; and some suggestion

### ISSUE THREAD

**Issue by LeMinhNgan** [roboflow/rf-detr#875]
### Search before asking

- [x] I have searched the RF-DETR issues and found no similar bug report.

### Bug

I find the new version miss the train/loss in progress bar. This information is quite important to me as the leading criterion. Could you please add it into the bar?

I also see the set of 3 learning_rate values in the proress bar: `lr, lr_min, lr_max`. It's very useful to understand whether the training is going to the end wall or not; and finetune the learning rate if I need to use its checkpoint as pretrained_weights for other model. However If any of them never change as the initial inputted value (I guess the lr_max), so that we should remove it save the progress bar space for other important indicators. Please consider

### Environment

- OS: window11
- RTX 5090
- Python 3.10

### Minimal Reproducible Example

```
parameters= {'dataset_dir': cfg.dataset_dir, 'epochs': cfg.epochs, 'batch_size': cfg.batch_size, 'grad_accum_steps': cfg.grad_accum_steps, 'lr': cfg.lr, 'progress_bar': cfg.progress_bar, 'aug_config': cfg.aug_config, 'checkpoint_interval': cfg.checkpoint_interval, 'output_dir': cfg.output_dir, 'early_stopping': False, 'run_test': False, 'resume': None}

model.train(**parameters)
    
```

### Are you willing to submit a PR?

- [ ] Yes, I'd like to help by submitting a PR!

**Comment by Borda** [comment]
@LeMinhNgan could you share what version you are using so we can try to reproduce it?

**Comment by LeMinhNgan** [comment]
@Borda 
it's version 1.6.1
I found the reason of the missing of train/loss in progress bar: if we use a screen whose width is small, the progress bar content is trimmed and then disappear.
We need to break the logged indicators into separated lines to avoid the phenomenon. Not sure if we can do that with pytorch-lightning progress bar.

Please find the attached image to see what i mean. Hope this one help you

<img width="1280" height="976" alt="Image" src="https://github.com/user-attachments/assets/ad338f5f-48ec-43e9-a34b-edba3ba778b9" />

**Comment by Borda** [comment]
@LeMinhNgan, you can also use "rich" progress bar, which shall have the warping automatically
